extensions/amp-ad-exit/0.1/amp-ad-exit.js

```diff
@@ -93,8 +105,26 @@ export class AmpAdExit extends AMP.BaseElement {
   exit(invocation) {
     const {args} = invocation;
     let {event} = invocation;
-    const target = this.targets_[args['target']];
-    userAssert(target, `Exit target not found: '${args['target']}'`);
+    let target;
+    if ('variable' in args) {
+      userAssert(
```

Comment:
Can we move this (and line 125 error message) to the beginning to provide a consistent error message? Otherwise, you will show `Variable target not found` if target not found.
```
userAssert(('variable' in args) != ('target' in args), `Only one of 'target' and 'variable' must be specified`);
    
```javascript

Reply:
Done. PTAL.